napari/plugins/_npe2.py

```diff
@@ -364,20 +423,34 @@ def _npe2_manifest_to_actions(
                     subitem = _npe2_submenu_to_app_model(item)
                     submenus.append((menu_id, subitem))
 
-    actions: List[Action] = [
-        Action(
-            id=cmd.id,
-            title=cmd.title,
-            category=cmd.category,
-            tooltip=cmd.short_title or cmd.title,
-            icon=cmd.icon,
-            enablement=cmd.enablement,
-            callback=cmd.python_name or '',
-            menus=cmds.get(cmd.id),
-            keybindings=[],
-        )
-        for cmd in mf.contributions.commands or ()
```

Comment:
Note I think the `or ()` would only be needed when `mf.contributions.commands` is `None` (will just return empty list if `mf.contributions.commands` is empty list) - thus have amended to the if/else below but happy to change.